bip-musig2/reference.py

```diff
@@ -109,7 +106,16 @@ def schnorr_verify(msg: bytes, pubkey: bytes, sig: bytes) -> bool:
 
 infinity = None
 
-def cbytes(P: Point) -> bytes:
+def bytes_from_point(P: Optional[Point]) -> bytes:
+    if is_infinite(P):
+        return bytes_from_int(0)
+    assert P is not None
+    return bytes_from_int(x(P))
+
+def cbytes(P: Optional[Point]) -> bytes:
```

Comment:
Nit: Should the version that handles infinity be a separate function (both to match the spec, and to avoid possible errors)?

(As well as `bytes_from_point` above)

Reply:
Yeah, I also wondered about this when I just looked at it again. Done.